### ISSUE TITLE
Firefox/NVDA HTML "region" summary of issues

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#33]
This is an issue in reference to #25 which is related to the ["region" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1145" alt="Screenshot of NVDA in test environment. Speech viewer for NVDA is focusing on one element and reads A graphical display of data Each element contains different information  region    Rect and Circle  grouping    graphic    3 Rect" src="https://user-images.githubusercontent.com/98043076/152585041-a2cbe960-6a55-40f7-8843-b149b0855c4b.png">

### Issue Summary
Issue 1: NVDA reads the HTML parent and first child grouping of the region when navigating to the starting graphic element. Similarly, the next grouping also does not work properly.
 
Issue 2: NVDA reads "blank" in the path graphic element. 

### Video of issue
Example 1 starts at 00:00:29
Example 2 starts at 00:00:36
https://user-images.githubusercontent.com/98043076/152573438-64098c9e-3b77-4efc-973f-56896db6f06b.mp4

### Steps to reproduce
Use NVDA "H" key to navigate to heading "html examples." Use the down arrow key to navigate into the HTML space. There is only one step (we would expect at least nine separate elements to navigate.)

### Expected behavior
We aren't actually testing HTML in this project, but if we are comparing the SVG to HTML elements in how they perform we need to note the issues found. 

### Technical Details
Firefox 96.0.3
NVDA 2021.3.1 
Windows 11 Build 22000.438
